docs/content/en/blog/news/pipecd-best-practice-01.md

```diff
@@ -0,0 +1,38 @@
+---
+date: 2021-12-29
+title: "PipeCD best practice 01 - operate your own PipeCD cluster"
+linkTitle: "PipeCD best practice 01"
+weight: 997
+description: "This blog is a part of PipeCD best practice series, a guideline for you to operate your own PipeCD cluster."
+author: Khanh Tran ([@khanhtc1202](https://twitter.com/khanhtc1202))
+---
+
+PipeCD is an open source project, you can freely use the released versions of PipeCD to create and operate a continuous delivery system for your service or company. [Quickstart](/docs/quickstart/) docs - a complete guide for you to install required components of PipeCD and deploy a simple application via PipeCD is a good point to get started. However, for the sake of the simplicity of that tutorial, some points to keep in mind when operating a PipeCD cluster have been omitted. You will review the PipeCD architecture again and get some tips on how to operate a PipeCD cluster in this post.
+
+### Again, review the PipeCD architecture
+
+![images](/images/architecture-overview-with-roles.png)
+<p style="text-align: center;">
+Component Architecture
+</p>
+
+> Note: Please refer to [concepts](/docs/concepts/) docs for definitions of PipeCD components such as Control Plane, Piped, Application, etc.
+
+At a glance, PipeCD is composed of 2 components, the Control Plane and the Piped(s). As the figure above, depending on your task/role, you need to work with different components of PipeCD.
+
+1. As a operator - platform team, most of your time working with PipeCD is to operate the PipeCD Control Plane. You may also need to cooperate with the product team, helping them install Piped(s) to their applications running cluster depending on your company team's structure. The space surrounded by a <span style="color: green;">green border</span> shows the operators working area.
+2. As a developer - product team, most of your time working with PipeCD is to control your applications' development via the PipeCD application configuration files. You may also need to cooperate with the platform team to install Piped(s) to your applications running cluster if necessary. The space surrounded by a <span style="color: blue;">blue border</span> shows the developers working area.
+
+### Tips to operate PipeCD cluster
+
+1. Only a single Control Plane needs to be installed and will be operated by the operators (platform team). The installed Control Plane doesn't have to be in the same cluster as your applications running cluster or the Piped installed cluster. Just make sure the Piped(s) can connect to the Control Plane via outbound requests is enough.
+
+2. Developers from the product team may need to self-install piped into their applications running cluster (due to some credential-related issues, if the platform team does not manage those credentials).\
```

Comment:
Is this the same with the number 3?

Reply:
Not really, what I want to mention in this (2) is that piped can be installed by platform team or developer team, while in (3) I want to mention that `we highly recommend running each Piped inside each cluster and that Piped will only manage the applications on that cluster` instead of using single piped to handle everything.

Reply:
> (due to some credential-related issues,

Somehow I want to avoid this expression, It could lead to a misunderstanding of our security issue.

Reply:
@khanhtc1202 Can you find a better sentence for this and we can merge and publish this blog.

Reply:
Sorry I missed the above message. Will update to avoid that.